src/status_im/ui/screens/profile/models.cljs

```diff
@@ -19,7 +19,7 @@
                         (re-frame/dispatch [callback-event base64]))
            on-error (fn [type error]
                       (.log js/console type error))]
-       (image-processing/img->base64 path on-success on-error 150 150)))
```

Comment:
why bigger?

Reply:
because of this comment by @errorists 
>> The identicon icon appears blurry, can we request an appropriate size to display here like 64px@2x so 128 Also, this was noticed by @hesterbruikman and now that she mentioned it I see it too, it appears the image is offset to the top left and not centred like it should?

on which I commented
> I think there has been conversations about this elsewhere (maybe captured in an issue too). The identicons were of size 40 - has always been; I increased it to 256 (4x - I think there are devices like that). Likewise, increased the size of Photo. Centered them as well.